packages/rn-tester-e2e/README.md

```diff
@@ -0,0 +1,80 @@
+# Building the app
+Bulding manually *.app* and *.apk* is required to run automation tests on local environment. 
+
+Before building app, make sure you ran:
+
+```bash
+cd react-native
+yarn install
+```
+## Building for iOS
+
+If you prevoiusly built RNTester, you may need to clean up build files and Pods:
+```bash
+rm -rf node_modules && yarn
+cd packages/rn-tester
+yarn clean-ios
+```
+
+Build the app for not M1 mac user: 
+
+1. Install Bundler `gem install bundler`. We use bundler to install the right version of CocoaPods locally.
+2. Install Bundler and CocoaPods dependencies: `bundle install && bundle exec pod install` or `yarn setup-ios-hermes`. In order to use JSC instead of Hermes engine, run: `USE_HERMES=0 bundle exec pod install` or `yarn setup-ios-jsc` instead.
+3. Open the generated `RNTesterPods.xcworkspace`.
+4. Build the app.
+
+If you are M1 mac user:
+1. Install ffi package `gem install ffi -v '1.15.5' --source 'https://rubygems.org/'`
+2. Install pods with new architecture, e.g. using JSC `USE_HERMES=0 arch -x86_64 pod install`
+3. Open the generated `RNTesterPods.xcworkspace`.
+4. Build the app.
+
+Find the **RNTester.app** in `~/Library/Developer/Xcode/DerivedData/RNTesterPods-{id}/Build/Products/Debug-iphonesimulator` and move the app to the following directory `/react-native/packages/rn-tester-e2e/apps`
```

Comment:
I think we can use `xcodebuild` from the command line and tell it to generate the app in some specific folder: [above listing 8 and 9](https://developer.apple.com/library/archive/technotes/tn2339/_index.html#//apple_ref/doc/uid/DTS40014588-CH1-HOW_DO_I_BUILD_MY_PROJECTS_FROM_THE_COMMAND_LINE_)